projects/ngneat/reactive-forms/src/lib/utils.ts

```diff
@@ -23,3 +23,23 @@ export function wrapIntoObservable<T>(value: T | Promise<T> | Observable<T>): Ob
 
   return of(value);
 }
+
+export function mergeErrors<E>(existing: Partial<E>, toAdd: Partial<E>) {
+  return existing
```

Comment:
```ts
return {
  ...existing,
  ...toAdd
}
```

Reply:
Thanks for the review
You're right: using spread operator on `null` or `undefined` will not throw so the the statement can be shortened.

On second thought, I also forgot a case in supporting `control.mergeErrors(null)`. 
If there was no error before, the resulting error should be `null` and not an empty object. 

i.e.

```typescript
export function mergeErrors<E>(existing: Partial<E>, toAdd: Partial<E>) {
  if (!existing && !toAdd) {
    return null;
  }
  return {
    ...existing,
    ...toAdd
  }
}
```

I'll update my PR and add a test case to cover this